safehttp/plugins/csp/csp_test.go

```diff
@@ -126,11 +126,13 @@ func TestBefore(t *testing.T) {
 		wantEnforcePolicy    []string
 		wantReportOnlyPolicy []string
 		wantNonce            string
+		wantXFO              []string
```

Comment:
Please add a test for multiple framing policies provided. I'd suspect that you need to alter what your PR does here to cover this case.

Reply:
We will probably have to look up the strictest one if we have more than one setting, which we currently don't have, so I guess it is fine for now.

Reply:
WDYM?

What if you create an interceptor with multiple framing policies?

Reply:
![image](https://user-images.githubusercontent.com/1256049/111293774-ec4aed00-8649-11eb-8baa-1f3be34d074e.png)
Why would anyone do that? And even if they did... all policies would have the same strictness value.

Reply:
Alright, I think I understand now what this PR is trying to do. See my PR-level comment.